python/cudf/cudf/pandas/scripts/run-pandas-tests.sh

```diff
@@ -193,9 +193,50 @@ and not test_numpy_ufuncs_basic[nullable_float-arctanh] \
 and not test_numpy_ufuncs_basic[nullable_float-deg2rad] \
 and not test_numpy_ufuncs_basic[nullable_float-rad2deg]"
 
+
+# TODO: Needs motoserver/moto container running on http://localhost:5000
```

Comment:
Do you think this is hard to setup in our CI?

Reply:
It's not too hard to specify in terms of GHA https://github.com/pandas-dev/pandas/blob/main/.github/workflows/unit-tests.yml#L89-L124

But I'm unsure if there's any differences with our custom runners. I can check this out in a follow up

Reply:
We can do that as a follow-up.